README.md

```diff
@@ -46,6 +46,18 @@ other tools for measuring time (which are non-deterministic and produce reports
 - Windows (WSL): `apt install time timeout`
 - macOS: `brew install gnu-time coreutils`
 
+If you would like to have autocomplete for `sinol-make` commands,
```

Comment:
```suggestion
### Autocompletion (optional)

If you would like to have autocompletion for `sinol-make` commands,
```

I'm putting it into another section to reduce the amount of needed installation steps. We don't want anyone to look into the installation steps, think "ugh, it's long, I don't wanna do so many steps" and not try our tool.